lib/ngsiUtils.js

```diff
@@ -22,10 +22,27 @@
  */
```

Comment:
CNR entry missing (I guess that referring to issue #104)

Reply:
Fixed in  2ba0df60b31a549ffe5b0e37723393b966c762a4